packages/docs/command-line-interface/exchange.md

```diff
@@ -4,16 +4,51 @@ description: Exchange Celo Dollars and Celo Gold via the stability mechanism
 
 ## Commands
 
+### Celo
```

Comment:
maybe add a note about the equivalence between `exchange:cgld` and `exchange:celo` in both docs?